collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -9,6 +9,7 @@
 import shutil
 import subprocess
 import sys
+import pandas as pd
```

Comment:
make sure to update the setup.py to include pandas

Reply:
removed, as not using it.